Minesweeper.py

```diff
@@ -99,6 +99,8 @@ def game_loop():
             if board[player_pos[1]][player_pos[0]] == 0:
                 board[player_pos[1]][player_pos[0]] = "F"
                 flags += 1
+            elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
```

Comment:
This condition is redundant since it is already checked in the previous statement. It should be removed to avoid unnecessary checks.
```diff
-             elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
+             # elif board[player_pos[1]][player_pos[0]] == "F":  # Redundant condition
```
---